File: collect.py
import csv

file_path = "data/data_with_classes.csv"

matclass1 = []
matclass2 = []
matclass3 = []

with open(file_path, 'r') as csvfile:
    reader = csv.DictReader(csvfile)
    for row in reader:
        label = int(row['Class'])
        if label == 1:
            matclass1.append(row)
        elif label == 2:
            matclass2.append(row)
        elif label == 3:
            matclass3.append(row)
        else:
            print("Unexpected label:", label)

print(matclass1[0])
print(matclass2[0])
print(matclass3[0])

File: collect.jl
using CSV
using DataFrames
#using PrettyTables

file_path="data/data_with_classes.csv"
df = CSV.read(file_path, DataFrame)

global matclass1=[]
global matclass2=[]
global matclass3=[]

for row in eachrow(df)
    
    class = row.Class  
    if class == 1
        push!(matclass1, row)
    elseif class == 2
        push!(matclass2, row)
    elseif class == 3
        push!(matclass3, row)
    else
        println("Unexpected label: ", class)
    end
end

println(matclass1[1])
println(matclass2[1])
println(matclass3[1])